lib/python/qmk/cli/kle2json.py

```diff
@@ -0,0 +1,93 @@
+"""Convert raw KLE to JSON
+
+"""
+import json
+import os
+from pathlib import Path
+from argparse import FileType
+from decimal import Decimal
+from collections import OrderedDict
+
+from milc import cli
+from kle2xy import KLE2xy
+
+from qmk.converter import kle2qmk
+
+
+class CustomJSONEncoder(json.JSONEncoder):
+    def default(self, obj):
+        try:
+            if isinstance(obj, Decimal):
+                if obj % 2 in (Decimal(0), Decimal(1)):
+                    return int(obj)
+                return float(obj)
+        except TypeError:
+            pass
+        return JSONEncoder.default(self, obj)
+
+
+@cli.argument('filename', nargs='*', help='The KLE raw txt to convert')
+@cli.argument('-f', '--force', action='store_true', help='Flag to overwrite current info.json')
+@cli.subcommand('Convert a KLE layout to a configurator JSON')
+def kle2json(cli):
+    """Convert a KLE layout to QMK's layout format.
+    """
+    if cli.args.filename:
+
+        # If filename is a path
+        if cli.args.filename[0][0] == "/" or cli.args.filename[0][0] == "./":
+            file_path = Path(cli.args.filename[0])
```

Comment:
(It'd be nice if github had multiline suggestions)

```suggestion
            file_path = Path(cli.args.filename)
```